test/wpool_SUITE.erl

```diff
@@ -104,12 +119,13 @@ too_much_overrun(_Config) ->
   TCPid ! {check, Worker, TaskId, 100}, % tiny runtime, to check
 
   ct:comment("Nothing happens..."),
-  receive
-  after 1000 -> ok
-  end,
+  _ = case get_messages(1000) of
```

Comment:
The code in the following (LINE, COL) locations has the same structure: (122, 5), (156, 5).